diff --git a/node_modules/@observablehq/inputs/dist/index.js b/node_modules/@observablehq/inputs/dist/index.js
index 04062e0..b8f45b0 100644
--- a/node_modules/@observablehq/inputs/dist/index.js
+++ b/node_modules/@observablehq/inputs/dist/index.js
@@ -1100,7 +1100,7 @@ function initialize(
 <style>${columns.map((column, i) => {
   const rules = [];
   if (align[column] != null) rules.push(`text-align:${align[column]}`);
-  if (width[column] != null) rules.push(`width:${length(width[column])}`);
+  if (width[column] != null) rules.push(`max-width:${length(width[column])}; max-width:${length(width[column])}`);
   if (rules.length) return `#${id} tr>:nth-child(${i + 2}){${rules.join(";")}}`;
 }).filter(identity).join("\n")}</style>`);
   function appendRows(i, j) {
