diff --git a/node_modules/@observablehq/inputs/dist/index.js b/node_modules/@observablehq/inputs/dist/index.js
index 04062e0..f8a947d 100644
--- a/node_modules/@observablehq/inputs/dist/index.js
+++ b/node_modules/@observablehq/inputs/dist/index.js
@@ -1100,7 +1100,7 @@ function initialize(
 <style>${columns.map((column, i) => {
   const rules = [];
   if (align[column] != null) rules.push(`text-align:${align[column]}`);
-  if (width[column] != null) rules.push(`width:${length(width[column])}`);
+  if (width[column] != null) rules.push(`width: max-width:${length(width[column])}; max-width:${length(width[column])}`);
   if (rules.length) return `#${id} tr>:nth-child(${i + 2}){${rules.join(";")}}`;
 }).filter(identity).join("\n")}</style>`);
   function appendRows(i, j) {
@@ -1519,3 +1519,4 @@ function eventof(input) {
 }
 
 export { bind, button, checkbox, color, date, datetime, disposal, email, fileOf, form, formatAuto, formatDate$1 as formatDate, formatLocaleAuto, formatLocaleNumber, formatNumber, formatTrim, input, number, password, radio, range, search, searchFilter, select, table, tel, text, textarea, toggle, url };
+console.log('YOOOOO')
\ No newline at end of file
